sections/page.liquid

```diff
@@ -16,14 +16,14 @@
 
 <div class="color-{{ section.settings.color_scheme }} gradient">
   <div class="page-width page-width--narrow section-{{ section.id }}-padding">
-    <h2 class="page-title {{ section.settings.heading_size }}">
+    <h2 class="page-title {{ section.settings.heading_size }}{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}">
       {%- if section.settings.page.title != blank -%}
         {{ section.settings.page.title | escape }}
       {%- else -%}
         {{ 'sections.page.title' | t }}
       {%- endif -%}
     </h2>
-    <div class="rte">
+    <div class="rte{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}">
```

Comment:
If the page is very long then the Intersection Observer never fires. This is the same bug @metamoni encountered on the product details page. Not sure how to fix it...

I'm starting to wonder if maybe instead of a threshold of `0.1` we should have a threshold of `0` but add a small delay to the animations.

Reply:
Yeah that sounds promising!

Reply:
K, ill get something up, `50px` seemed pretty good when I just tested 👍

Reply:
👍 I'll merge this one, since we're tackling that elsewhere.